inffast.c

```diff
@@ -156,15 +156,29 @@ void Z_INTERNAL zng_inflate_fast(PREFIX3(stream) *strm, unsigned long start) {
        window is overwritten then future matches with far distances will fail to copy correctly. */
     extra_safe = (wsize != 0 && out >= window && out + INFLATE_FAST_MIN_LEFT <= window + wsize);
 
+#define REFILL() do { \
+        hold |= load_64_bits(in, bits); \
+        in += 7; \
+        uint64_t tmp = ((bits >> 3) & 7); \
```

Comment:
So we need this `tmp` variable for anything anymore?

Reply:
Removing it has had measurable performance impact at various points in my past experiments, so I'd left it out of superstition, but I've removed it for now in favour of being more methodical about weird, compiler-specific hacks later.